packages/mui-material-next/migration.md

```diff
@@ -156,6 +164,141 @@ The `inputComponent` is deprecated in favor of `slots.input`:
  />
 ```
 
+## Chip
+
+### Removed the "default" color option
+
+The `"default"` option is no longer accepted for the color prop.
+
+### Removed combined styleOverrides keys
+
+The following `styleOverrides` `MuiChip` keys were removed:
+
+- `clickableColorPrimary`
+- `clickableColorSecondary`
+- `deletableColorPrimary`
+- `deletableColorSecondary`
+- `outlinedPrimary`
+- `outlinedSecondary`
+- `filledPrimary`
+- `filledSecondary`
+- `avatarSmall`
+- `avatarMedium`
+- `avatarColorPrimary`
+- `avatarColorSecondary`
+- `iconSmall`
+- `iconMedium`
+- `iconColorPrimary`
+- `iconColorSecondary`
+- `labelSmall`
+- `labelMedium`
+- `deleteIconSmall`
+- `deleteIconMedium`
+- `deleteIconColorPrimary`
+- `deleteIconColorSecondary`
+- `deleteIconOutlinedColorPrimary`
+- `deleteIconOutlinedColorSecondary`
+- `deleteIconFilledColorPrimary`
+- `deleteIconFilledColorSecondary`
+
+You can replace them by combining other keys with CSS selectors inside the `MuiChip` `styleOverrides` key.
+The following example replaces the usage of `filledPrimary` with the `filled` key and a CSS selector for the `MuiChip-colorPrimary` class
+
+```diff
+ const theme = extendTheme({
+   components: {
+     MuiBadge: {
+       styleOverrides: {
+-       filledPrimary: {
++       filled: {
++         "&.MuiChip-colorPrimary": {
```

Comment:
Should we instead add example using the `variants` API? It will likely be the API that will stick with the zero-runtime CSS in JS.

Reply:
Done 👌🏼